SUMMARY.md

```diff
@@ -17,6 +17,7 @@
 * [Strategies](strategies/README.md)
   * [What is a strategy?](strategies/what-is-a-strategy.md)
   * [Create a new strategy](strategies/create.md)
+  * [How to write a basic strategy](strategies/how-to-write-a-basic-strategy.md)
```

Comment:
Isn't "Create a new strategy" having the same purpose than "How to write a basic strategy"? Shouldnt this be merged together?

Reply:
Indeed, kinda hesitated on this and ended up thinking it would maybe be too big... Do you want me to merge them?